SofascoreAcademy/SofascoreAcademy/Views/MainViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  MainViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 08.04.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+protocol parentSportSlugPicker {
+    func getPressedTab(request: sport_slug?)
+}
+
+protocol settingsButtonTap {
+    func reactToSetingsTap()
```

Comment:
Isto naming koji bi bio vise u duhu swift jezika, settingsTapped ili jos bolje didSettingsTap. Jer on njemu javlja da se to dogodilo i onda je na VC da odluci oce li react-at ili ne.